packages/utils/src/lib/git.ts

```diff
@@ -37,36 +37,66 @@ export async function toGitPath(
   return formatGitPath(path, gitRoot);
 }
 
+export class GitStatusError extends Error {
```

Comment:
The class contains a lot of logic, so I would suggest it should be covered by unit tests (as suggested by our GitHub Action's annotations :grin:).

![image](https://github.com/code-pushup/cli/assets/34691111/ffac0591-b93d-49a3-b1bd-4facdb309c2e)

Reply:
How beautiful this comments are! 
🔥 😍 🦚

Reply:
Done

Reply:
I don't see the new tests :confused:

Reply:
It was gone for some reason... added the test now.